include/bluetooth/hci.h

```diff
@@ -564,6 +564,20 @@ struct bt_hci_rp_write_auth_payload_timeout {
 	uint16_t handle;
 } __packed;
 
+#define BT_DATA_PATH_DIRECTION_INPUT            0x00
+#define BT_DATA_PATH_DIRECTION_OUTPUT           0x01
```

Comment:
There is some overlap here with #29947. I'll leave it to other reviewers to judge which variant they prefer, but these PRs will have to be rebased on top of each other.

Reply:
I removed the two defines